mods/ra/rules/misc.yaml

```diff
@@ -126,10 +126,6 @@ CAMERA:
 	RevealsShroud:
 		Range: 10c0
 		Type: CenterPosition
-	ProximityCaptor:
-		Types: Camera
-	DetectCloaked:
-		Range: 10c0
```

Comment:
This was actually on purpose, see here: fc2c46a2c25aad0c5c05a869d18e37762f673fe8

But cameras are used in too many places now for this to not potentially have bad side effects.